lib/Github/Api/Repo.php

```diff
@@ -82,6 +82,21 @@ public function activity($username, $repository)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/stats/commit_activity');
     }
 
+    /**
+     * Get the contents of the repository's license file, if one is detected.
```

Comment:
there is no comma before if.

Reply:
I don't understand the comment here? But it's a exact copy of description from the github api docs, so ok for me!

Reply:
Just a grammar tip.

Reply:
As @acrobat mentioned, I've just copied the definition from the official docs:
https://developer.github.com/v3/licenses/#get-the-contents-of-a-repositorys-license

Reply:
Interesting. The comma is only valid if the sentence starts with the if clause.
But then never mind :)

Reply:
The English language doesn't really have strict rules on commas. In general, if there's a pause when you read it, you're allowed to put in a comma. :)

Reply:
https://www.google.com/amp/s/dictionary.cambridge.org/de/amp/britisch-grammatik/conditionals
I am talking about normal rules. Of course you can still do whatever you want^^